examples/istioctl-analyze/README.md

```diff
@@ -0,0 +1,37 @@
+# Istioctl Analyze
+
+The `istioctl-analyze` config function detects potential issues with your Istio
+configuration and output errors as results. This example invokes the
+istioctl-analyze function using declarative configuration.
+
+## Function invocation
+
+The function is invoked using the function configuration in
+`functions/fn-config.yaml`.
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/istioctl-analyze .
+kpt fn run istioctl-analyze/configs --fn-path istioctl-analyze/functions --results-dir istioctl-analyze/results
```

Comment:
How do I configure this function? e.g. providing kubeconfig for the cluster to hit?

Reply:
This function is meant to be run on local configs. I will work on providing a kubeconfig in the kubeval blueprint.